source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,362 @@
+3. Core Networking would be the next to run terraform on, checking the code and modules it uses for region references and changing them.
+
+
+**With the infrastructure in London no longer available, we would use the original ranges**
+
+
+### Transit Gateway
+
+
+To rebuild the transit gateway in a new region, first find the code, located here, in core-network-services. https://github.com/ministryofjustice/modernisation-platform/tree/main/terraform/environments/core-network-services
+
+
+Using the admin role, run a terraform plan and apply. This hasnt been done from scratch since the platforms creation, so expect to debug issues.
+
+
+### Firewall
+
+
+We make use of AWS Network Firewalls in two ways. First, we use a centralised inspection VPC to control traffic entering and exiting the Modernisation Platform to and from internal Ministry of Justice services. Second, we use inline-inspection in our egress VPCs to control traffic exiting the Modernisation Platform to the internet.
+
+
+Located in the Core-Network-Services account, DNS, networking and other network related infrastructure will be down.
+
+
+The firewall rules can be located here - https://github.com/ministryofjustice/modernisation-platform/tree/main/terraform/environments/core-network-services/firewall-rules
+
+
+Live data & non-live data.
+
+
+Peer the new Transit gateway build in eu-west-1 to the MOJ transit gateway that is in eu-west-2.
+
+
+### Account dependencies
+
+
+|Account|Information|
+|----------------- |------------------------------------------------ |
+|Core-Shared-Services|Creates the core shared services account resources, AMIs, KMS, S3 and other shared services customers require are based in here. |
+|Core-Network-Services | Creates the core networking account resources, and contains the transit gateway terraform, as well as firewall rules. |
+|Core-Logging|Creates the core logging account resources, not essential for operational function but a part of the platform. As well as for troubleshooting, logging is required for audit purposes and therefore should also be prioritised. |
+|Core-Security|Creates the core security account resources |
+|Cloud-Platform-Transit-Gateways|Across all regions. |
+|Core-VPC-*|Creates the core VPC resources in the VPC accounts |
+|Modernisation-Platform | Creates key resources such as S3 state buckets for the platform |
+
+Once terraform has been run against these accounts, member accounts can be next, as technically, all that is required for the platform should now exist.
+
+### Member Accounts 
+
+After the networking has been completed, the core accounts created, and the modernisation platform itself built, the next step is creating member-accounts. Start with cooker, example and sprinkler, building from the mod-platform repo and then applying the terraform in the environments repo. This will allow us to test the steps required for our customers to follow when recreating their infrastructure from the environments repo.
```

Comment:
`After the networking has been completed, the core accounts created, and the modernisation platform itself built, the next step is creating member-accounts. Start with cooker, example and sprinkler, building from the mod-platform repo and then applying the Terraform in the environments repo. This will allow us to test the steps required for our customers to follow when recreating their infrastructure from the environments repo.`